examples/simpleserver/src/main/java/com/symphony/simpleserver/smb/api/Parser.java

```diff
@@ -377,17 +380,38 @@ public SmbEndpointDescription makeEndpointDescription(SessionDescription sdpAnsw
 
             } else if (mediaDescription.type == MediaDescription.Type.VIDEO) {
                 final var video = new SmbVideo();
-
-                video.ssrcs = new ArrayList<>();
-                mediaDescription.ssrcs.forEach(ssrc -> video.ssrcs.add(ssrc.ssrc));
-                video.ssrcGroups = new ArrayList<>();
-                mediaDescription.ssrcGroups.forEach(ssrcGroup -> {
-                    final var smbSsrcGroup = new SmbSsrcGroup();
-                    smbSsrcGroup.ssrcs = new ArrayList<>(ssrcGroup.ssrcs);
-                    smbSsrcGroup.semantics = ssrcGroup.semantics;
-                    video.ssrcGroups.add(smbSsrcGroup);
+                final var streamsMap = new HashMap<String, SmbVideoStream>();
+
+                mediaDescription.ssrcs.forEach(ssrc -> {
+                    final var smbVideoStream = streamsMap.computeIfAbsent(
+                            ssrc.mslabel,
+                            key -> {
+                                final var value = new SmbVideoStream();
+                                value.id = ssrc.mslabel;
+                                value.content = "video";
```

Comment:
[Minor] Probably we don't support screensharing on simpleclient, but I guess we can handle it on simple server as example how it should be handled

```suggestion
                                value.content = "slides".equals(mediaDescription.content)?  "slides" : "video";
```